src/components/SiteLayout.tsx

```diff
@@ -82,6 +82,10 @@ const SiteLayout: FC<Props> = ({ children }) => {
   useEffect(() => {
     const accessToken = Cookies.get('accessToken')
     const refreshToken = Cookies.get('refreshToken')
+    const storedUserState = localStorage.getItem('lenster.store' || '{}')
```

Comment:
**question:** @0xkeivin we can get this value from the store `usePersistStore` right? any reason we are using localstorage?

Reply:
have overlooked that. 😓 
let me try replacing it with values from `useAppPersistStore`

Reply:
have updated in latest commit